src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -73,4 +119,322 @@ public void userSeesDownloadFiles(final String expectedLength) {
         Select listBox = new Select(selectElement);
         assertEquals(listBox.getOptions().size(), Integer.parseInt(expectedLength));
     }
+    /**
+     * Select 2015 edition products file from drop down and download .xml file.
+     * @throws Exception - to handle IOException with FileUtil
+     */
+
+    @When("^I download the 2015 edition products file$")
+    public void download2015editionProductsFile() throws Exception {
+        /**
+         * Clean all files from download directory so each time there is only latest
+         * download file to read.
+         */
+        this.movefiles();
+        FileUtils.cleanDirectory(dir);
```

Comment:
This step has to happen before each of these "@When"s, right? Maybe it would make more sense for it to be in the @Given step?

Reply:
No. I just wanted to move files only once in the beginning to copy any existing files in the folder, so we don't loose them before cleaning them from the directory and also so that the directory is empty before download process begins. It is not needed for later test runs.

Reply:
Oh, I see what you mean. In that case, this is actually not a good thing. What if someone skips this test? Or they run out of order? I don't think there's a guarantee that all of the Scenarios run in the order they're in the file. Would it make more sense for the `this.movefiles();` command to be in a **Background:** section? That way it _will_ run once before all of the Scenarios, but only once.

Then maybe the `FileUtils.cleanDirectory(dir);` call could be in the `@Given I am on the CHPL Download page` just one time, but it'd run at the beginning of each Scenario.